lib/standard/stream.nit

```diff
@@ -392,6 +392,34 @@ abstract class BufferedReader
 		return c.ascii
 	end
 
+	# Peeks up to `n` bytes in the buffer, returns an empty string on EOF
+	#
+	# The operation does not consume the buffer
+	#
+	# ~~~nitish
+	# 	var x = new FileReader("File.txt")
+	#	assert x.peek(5) == x.read(5)
+	# ~~~
```

Comment:
Maybe this can be an executable test?

Something like
~~~nit
var file = "test.txt"
var fw = new FileWriter.open(file)
fw.write("HelloWorld")
fw.close

var fr = new FileReader(file)
assert fr.peek(5) == "Hello"
assert fr.peek(200) == "World"
assert fr.peek(5) == ""
fr.close

file.file_delete
~~~